.github/workflows/ci.yml

```diff
@@ -33,44 +33,34 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 6.15.1
+
       - name: Set node version to ${{ matrix.node_version }}
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node_version }}
+          cache: "pnpm"
```

Comment:
The cache function of this action doesn't respect different node versions. This could result in problems, could it?
At least this was the reason why we did use the cache action.

Reply:
asked about that at setup node a bit back.works bc its caching the pnpm store,  not node_modules after install

https://github.com/actions/setup-node/issues/304#issuecomment-886241508

Reply:
Good to know! This let's me thing about to slowly also start switching away from yarn 🤔